test/types/models/User.ts

```diff
@@ -117,7 +117,7 @@ User.addHook('afterDestroy', async (instance, options) => {
 // Model#addScope
 User.addScope('withoutFirstName', {
   where: {
-    firstName: {
+    lastName: {
```

Comment:
I don't remember but it's weird, I'll take a look at the commit history and maybe revert

Reply:
Can't find a good reason for this change. It was probably an accident, I've reverted it.

Reply:
Update: my IDE didn't update its error list fast enough for me to notice but there was a reason for this change :)

`firstName: { [Op.is]: null }` is flagged as an error because `firstName` is not nullable.
`lastName` is nullable so I used it instead